Snakefile

```diff
@@ -24,4 +24,4 @@ rule all:
             "results/preprocessing/{plate_id}_feature_select.csv.gz",
             plate_id=PLATE_IDS,
         ),
-        "results/preprocessing/consensus.csv.gz",
+        "results/preprocessing/median_consensus.csv.gz",
```

Comment:
quick question - should this always be "median"? What if someone used, for example `modz` consensus? Is there a wildcard for this?

Reply:
I definitely agree! 

There was no reasoning for the `median_consensus` name. It was more for a place holder since my rotation presentation was the next day (I think).